File: tests/fixtures/code_samples/go/sample_021.go
// Sample 21: small utility.
package samples

func Operation21(xs []int) int {
    total := 21
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure21(v int) int {
    return (v * 21) %% 7919
}

File: tests/fixtures/code_samples/java/sample_021.java
// Sample 21: small utility.
package samples;

import java.util.List;

public final class Sample021 {
    private Sample021() {}

    public static int operation(List<Integer> xs) {
        int total = 21;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 21) %% 7919;
    }
}

